config/sample-idex.cfg

```diff
@@ -44,13 +50,21 @@ gcode:
 
 # Activate the primary extruder
 [gcode_macro T0]
+variable_offset_applied: 0
 gcode:
-    PARK_{printer.toolhead.extruder}
+    {% set svv = printer.save_variables.variables %}
+    {% if "x" in printer.toolhead.homed_axes %} # this check ensures compatibility with Cura
+        PARK_extruder1
```

Comment:
I prefer the `PARK_{printer.toolhead.extruder}` over this, as that will scale to more than 2 extruders if you have such a setup. (e.g. this docs can still be useful for people doing toolchangers).

Reply:
I will switch back when I go through this variableifying everything. I've been hesitant to so far however because they don't always seem to work the way I expect. For example, I wanted to use the axis.maximum value to park the right extruder, but even when eh dual_carraige is enabled, the axis.maximum value is that of the primary carriage.